src/client/index.ts

```diff
@@ -128,6 +133,13 @@ export class MonoClient<
     }
 
     if (attempt + 1 < maxAttempt && this.shouldRetry(request, response)) {
+      if (
```

Comment:
you can simplify this be doing something like
`this.config.retry?.delayInSeconds ?? 0 > 0`

Reply:
Get type error. `Object is possibly 'undefined'`